File: SDKs/amazonaws.com/elasticloadbalancing/2012-06-01/java/lib/src/main/java/openapisdk/models/shared/DescribeLoadBalancerPolicyTypesOutput.java
package openapisdk.models.shared;



/**
 * DescribeLoadBalancerPolicyTypesOutput
 * Contains the output of DescribeLoadBalancerPolicyTypes.
**/
public class DescribeLoadBalancerPolicyTypesOutput {
    public PolicyTypeDescription[] policyTypeDescriptions;
    public DescribeLoadBalancerPolicyTypesOutput withPolicyTypeDescriptions(PolicyTypeDescription[] policyTypeDescriptions) {
        this.policyTypeDescriptions = policyTypeDescriptions;
        return this;
    }
}

File: SDKs/amazonaws.com/elasticloadbalancing/2012-06-01/python/src/sdk/models/shared/describeloadbalancerpolicytypesoutput.py
import dataclasses
from typing import Optional
from ..shared import policytypedescription as shared_policytypedescription


@dataclasses.dataclass
class DescribeLoadBalancerPolicyTypesOutput:
    r"""DescribeLoadBalancerPolicyTypesOutput
    Contains the output of DescribeLoadBalancerPolicyTypes.
    """
    
    policy_type_descriptions: Optional[list[shared_policytypedescription.PolicyTypeDescription]] = dataclasses.field(default=None)
    
